ci_pyproject.toml

```diff
@@ -16,7 +16,7 @@ markers = [
 testpaths = [
     "tests"
 ]
-addopts = "-vs --ci --reruns 2 --reruns-delay 1 -m 'not incident and not unstable' --html=artifacts/report.html"
+addopts = "-vs --ci --run-headless --reruns 2 --reruns-delay 1 -m 'not incident and not unstable and not headed' --html=artifacts/report.html"
```

Comment:
so currently there is no set of flags for the headed runs? or do these flags just add to the base command when we are running typical test runs?

Reply:
All addopts are dynamically added to pytest runs when you call them from the command line. This change removes `headless` as a default option for `ci` and creates a new pyproject file for headed runs, see the CI workflow yamls